src/notification/LeaveNotification.js

```diff
@@ -0,0 +1,39 @@
+import { RichEmbed, Guild } from "discord.js";
+import GrouperClient from "../structures/GrouperClient";
+import Notification from './Notification.js'
+
+export default class LeaveNotification extends Notification {
+    /**
+     * Creates a new notification handler for when guilds are left
+     * 
+     * @param {GrouperClient} client 
+     * @param {String} guild - Support server snowflake
+     * @param {String} channel - Alert channel snowflake
+     */
+    constructor(client, guild, channel) {
+        super(client, guild, channel);
```

Comment:
I'm pretty sure that this constructor is redundant as it will pass any props passed to the parent by default. Might be wrong though! 😛